tests/torchtune/config/test_instantiate.py

```diff
@@ -74,3 +78,53 @@ def test_instantiate(self, config, module):
         del config.test.dim
         actual = instantiate(config.test, 3)
         assert self.get_dim(actual) == 3
+
+
+ASSETS = Path(__file__).parent.parent.parent / "assets"
+
+
+class TestInstantiateTokenizer:
```

Comment:
Appreciate you adding this test, but actually don't think this is necessary. This is simply testing that an object can be instantiated and has the correct parameters, but this is already covered by the other tests in this directory

Reply:
I added the test to mostly check for the null case, which we couldn't do with `RMSNorm`.

In [`4eeb1fd` (#1363)](https://github.com/pytorch/torchtune/pull/1363/commits/4eeb1fd99d6e54335ce492b7023233b764a2e6a1), I simplified the test to only check the null case.